mensor/measures/types.py

```diff
@@ -692,8 +691,8 @@ def desc(self):
 
 class _Dimension(_ProvidedFeature):
 
-    def __init__(self, name, expr=None, default=None, desc=None, shared=False, partition=False, requires_constraint=False, provider=None):
-        _ProvidedFeature.__init__(self, name, expr=expr, default=default, desc=desc, shared=shared, provider=provider)
+    def __init__(self, name, expr=None, default=None, desc=None, shared=False, partition=False, requires_constraint=False, provider=None, **attrs):
+        _ProvidedFeature.__init__(self, name, expr=expr, default=default, desc=desc, shared=shared, provider=provider, **attrs)
```

Comment:
as discussed, perhaps we should be more explicit with which attrs get sent through

Reply:
👍